### PR TITLE
Feature/kubernetes

### PR DESCRIPTION
**Pull request by Mhm0ud**
This is the minimal k8s configuration needed to run the backend services.

### CODE REVIEW

**Comment by icebob** [comment]
Wow, nice job!

File: kubernetes/api-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+# Service for Moleculer API Gateway service
+#########################################################
+apiVersion: v1
+kind: Service
+metadata:
+  name: api
+  namespace: default
+  labels:
+    name: api
+    app: kantav
```
Comment (icebob) [inline kubernetes/api-deployment.yaml:11]: typo

File: kubernetes/cards-attachments-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        - name: cards-attachments
+          image: kantab
+          envFrom:
+            - configMapRef:
+                name: kantab-configmap
+            - secretRef:
+                name: kantab-secrets
+          env:
+            - name: SERVICES
+              value: cards.checklists
```
Comment (icebob) [inline kubernetes/cards-attachments-deployment.yaml:32]: wrong service name

File: kubernetes/cards-deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        - name: cards
+          image: kantab
+          envFrom:
+            - configMapRef:
+                name: kantab-configmap
+            - secretRef:
+                name: kantab-secrets
+          env:
+            - name: SERVICES
+              value: cards
```
Comment (icebob) [inline kubernetes/cards-deployment.yaml:32]: wrong service name

Comment (Mhm0ud) [reply kubernetes/cards-deployment.yaml:32]: There is a service name 'cards'

Comment (icebob) [reply kubernetes/cards-deployment.yaml:32]: Sorry, I overlooked it.

**Pull request merged**